include/geom/intersection_tools.h

```diff
@@ -69,6 +69,19 @@ WithinSegmentResult within_segment(const Point & s1,
                                    const Point & p,
                                    const Real tol = TOLERANCE);
 
+/**
+ * Checks whether or not the given points are collinear
+ * @param p1 The first point
+ * @param p2 The second point
+ * @param p3 The third point
+ * @param tol The tolerance to use
```

Comment:
Here's another one where tol isn't specified as relative or absolute and where we're not worrying about units.

Here it's clearer that performance might be an issue (though I bet with `norm_sq()` you'd be fine) but I'd still prefer a relative tolerance and I'd still insist that we document an absolute tolerance.  I probably sound like an ass about this, but really I should be sounding like an ass and rattling heavy chains: I feel like the Ghost Of Screwed Up Displaced Mesh Problems Past.